lib/model/folder.go

```diff
@@ -449,6 +449,12 @@ func (f *folder) scanSubdirs(subDirs []string) error {
 		return err
 	}
 
+	select {
+	case <-f.ctx.Done():
+		return f.ctx.Err()
+	default:
```

Comment:
There exists check like this in the snap.WithPrefix..., does this one actually help?

Reply:
Trying to spell out the reason why I added this all arguments dissolved and they only remaining reason is that a panic on sentry happened at this point, which is not an argument at all. Rechecking the goroutines shows the problem is with lib/connections -> removing.